org.musicbrainz.Picard.json

```diff
@@ -4,8 +4,6 @@
     "runtime-version": "5.11",
     "sdk": "org.kde.Sdk",
     "command": "picard",
-    "rename-desktop-file": "picard.desktop",
```

Comment:
Has the desktop file also been renamed?

Last I checked they were adding an appdata and renaming the icons only.

Reply:
They have.

Reply:
Great!